graphjet-core/src/main/java/com/twitter/graphjet/algorithms/counting/tweet/TopSecondDegreeByCountTweetMetadataRecsGenerator.java

```diff
@@ -46,12 +46,13 @@ private static void addToSocialProof(
     int maxTweetSocialProofSize
   ) {
     SmallArrayBasedLongToDoubleMap[] socialProofsByType = nodeInfo.getSocialProofs();
+    TweetIDMask tweetIDMask = new TweetIDMask();
```

Comment:
this function is being called hundreds of thousand times a second. this additional object creation will be roughly 1M bytes per second. why not use static function? or at least this should be a class variable?